test/verify/check-apps

```diff
@@ -162,14 +170,18 @@ class TestApps(packagelib.PackageCase):
 
         # unknown OS: nothing gets installed
         m.write("/etc/os-release", 'ID="unmapped"\nID_LIKE="mysterious"\nVERSION_ID="1"\n')
+        b.reload()
+        b.enter_page('/apps')
         b.click("#refresh")
         # the progress bar is too fast to reliably catch it
-        time.sleep(1)
-        b.wait_not_present("#refresh-progress")
-        b.wait_visible(".pf-v5-c-empty-state")
+        with b.wait_timeout(1):
```

Comment:
Timeouts of 1s are way too aggressive for our CI. Please keep this part of the code as it was. It's not pretty, but I don't see a better way of doing that.